ray-operator/config/samples/vllm/ray-service.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: llama-3-8b
+spec:
+  serveConfigV2: |
+    applications:
+    - name: llm
+      route_prefix: /
+      import_path:  ray-operator.config.samples.vllm.serve:model
+      deployments:
+      - name: VLLMDeployment
+        num_replicas: 1
+        ray_actor_options:
+          num_cpus: 8
+          # NOTE: num_gpus is set automatically based on TENSOR_PARALLELISM
+      runtime_env:
+        working_dir: "https://github.com/ray-project/kuberay/archive/master.zip"
+        pip: ["vllm==0.4.3"]
+        env_vars:
+          MODEL_ID: "meta-llama/Meta-Llama-3-8B-Instruct"
+          TENSOR_PARALLELISM: "2"
+  rayClusterConfig:
+    headGroupSpec:
+      rayStartParams:
+        dashboard-host: '0.0.0.0'
+      template:
+        spec:
+          containers:
+          - name: ray-head
+            image: rayproject/ray-ml:2.33.0.914af0-py311
```

Comment:
I am not sure what 914af01 is. I guess it is the commit SHA. Maybe we can use 2.34.0-py311 instead?

Reply:
I'm not sure if the tagging for the ray-ml image changed recently. But 2.34.0-py311 does not exist, but there is 2.34.0.deprecated-py311 and  2.34.0.fc8721-py311, any suggestions on which one to use?

Reply:
(same applies for 2.33)

Reply:
It it helps: https://hub.docker.com/r/rayproject/ray-ml/tags?page=&page_size=&ordering=&name=2.34

Reply:
I checked internally and found that we plan to deprecate `ray-ml` images. I think we don't need to change the image now. We should update the image in the future when the Ray community has a new plan for Ray images with ML dependencies. Would you mind adding a comment about this?

Reply:
I think it will be noisy as a comment in the example YAML. I opened an issue to track it instead https://github.com/ray-project/kuberay/issues/2292